src/domain/event/validation.ts

```diff
@@ -426,6 +434,92 @@ export const draftEventSchema = Yup.object().shape(
   CYCLIC_DEPENDENCIES
 );
 
+export const getExternalUserEventSchema = (
```

Comment:
There are quite many conditional validation rules. You could add unit tests to cover those cases.

Reply:
Added unit tests: [330dc3d](https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/330dc3dcde8d7a77badad592a10e94c7e3751d13)